app/grandchallenge/algorithms/tasks.py

```diff
@@ -129,7 +134,7 @@ def on_job_creation_error(self, task_id, *args, **kwargs):
 def execute_algorithm_job_for_inputs(*, job_pk):
     job = Job.objects.get(pk=job_pk)
 
-    # Send an email to the algorithm editors and creator on job failure
+    # Notify the job creator failure
```

Comment:
So `if isinstance(res, ImageImportError):` a couple of lines above works now?

Reply:
Don't know, didn't test it, existing bug if not so will follow that up in a later PR if necessary.